controllers/funding/views/past-grants.njk

```diff
@@ -0,0 +1,129 @@
+                {{ breadcrumbTrail(breadcrumbs) }}
+
+                {% if not enableNewPastGrantsSearch %}
+
+                    <section class="s-prose u-constrained-content-wide">
+                        <p>{{ copy.intro }}</p>
+                        {{ inlineFeedback(
+                            description = "Past Grants",
+                            promptLabel = copy.survey.prompt,
+                            fieldLabel = copy.survey.label
+                        )  }}
+                        <hr />
+                        <p>{{ copy.optionsIntro }}</p>
+                    </section>
+
+                    <ul class="flex-grid">
+                        <li class="flex-grid__item">
+                            <div class="card">
+                                <header class="card__header">
+                                    <h2 class="card__title">{{ copy.grantnav.title }}</h2>
+                                </header>
+                                <div class="card__body">
+                                    <p><a
+                                        data-ga-on="click"
+                                        data-ga-event-category="Past grants"
+                                        data-ga-event-action="GrantNav"
+                                        class="btn btn--medium"
+                                        href="{{ grantNavLink }}">
+                                        {{ copy.grantnav.linkText }}
+                                    </a><p>
+                                    <p>{{ copy.grantnav.subtitle }}</p>
+                                </div>
+                            </div>
+                        </li>
+                        <li class="flex-grid__item">
+                            <div class="card">
+                                <header class="card__header">
+                                    <h2 class="card__title">{{ copy.legacy.title }}</h2>
+                                </header>
+                                <div class="card__body">
+                                    <p><a
+                                        data-ga-on="click"
+                                        data-ga-event-category="Past grants"
+                                        data-ga-event-action="Legacy past grants"
+                                        class="btn btn--medium"
+                                        href="{{ localify('/funding/search-past-grants') }}">
+                                        {{ copy.legacy.linkText }}
+                                    </a></p>
+                                </div>
+                            </div>
+                        </li>
+                    </ul>
+
+                {% else %}
+                    <section class="s-prose u-constrained-content-wide">
+                        {{ __('funding.pastGrants.intro', grantNavLink) | safe }}
+                    </section>
+
+                    <form class="grants-searchbar"
```

Comment:
Guessing we should make this a component at some point?

Reply:
I did think this, but then the other version of it uses the JS-powered one by default, which we didn't need here.